easy_thumbnails/engine.py

```diff
@@ -57,6 +62,9 @@ def save_image(image, destination=None, filename=None, **options):
                 max(image.size) >= settings.THUMBNAIL_PROGRESSIVE):
             options['progressive'] = True
         try:
+            icc_profile = image.info.get('icc_profile')
+            if icc_profile and 'keep_icc_profile' in thumbnail_options:
```

Comment:
I think change to `thumbnail_options.get('keep_icc_profile')` - the value might be set to `False`.